tutorials/setup.md

```diff
@@ -0,0 +1,205 @@
+
+If using an older release or a long-term support release, be careful to double-check that the version of `runc` is new enough (running `runc --version` should produce `spec: 1.0.0`), or else build your own.
+
+Be careful to double-check that the version of golang is new enough, version 1.8.x or higher is required.  If needed, golang kits are available at https://golang.org/dl/
+
+### Get Source Code
+
+Clone the source code using:
+
+```bash
+git clone https://github.com/cri-o/cri-o # or your fork
+cd cri-o
+```
+
+If you want a specific version, you can use
+```
+git checkout release-$(RELEASE-VERSION)
+```
+to get that version.
+
+For instance, if you want to be compatible with the latest kubernetes release, RELEASE-VERSION=1.14
+
+### Build
+
+To install with default buildtags using seccomp, use:
+
+```bash
+make install.tools
+make
+sudo make install
+```
+
+Otherwise, if you do not want to build `CRI-O` with seccomp support you can add `BUILDTAGS=""` when running make.
+
+```bash
+make install.tools
+make BUILDTAGS=""
+sudo make install
+```
+
+#### Build Tags
+
+`CRI-O` supports optional build tags for compiling support of various features.
+To add build tags to the make option the `BUILDTAGS` variable must be set.
+
+```bash
+make BUILDTAGS='seccomp apparmor'
+```
+
+| Build Tag | Feature                            | Dependency  |
+|-----------|------------------------------------|-------------|
+| seccomp   | syscall filtering                  | libseccomp  |
+| selinux   | selinux process and mount labeling | libselinux  |
+| apparmor  | apparmor profile support           | <none>      |
+
+### Setup CNI networking
+
+A proper description of setting up CNI networking is given in the
+[`contrib/cni` README](contrib/cni/README.md). But the gist is that you need to
+have some basic network configurations enabled and CNI plugins installed on
+your system.
+
+### CRI-O configuration
+
+If you are installing for the first time, generate and install configuration files with:
+
+```
+sudo make install.config
+```
+
+#### Validate registries option in /etc/crio/crio.conf
+
+Edit `/etc/crio/crio.conf` and verify that the registries option has valid values in it.  For example:
+
+```
+registries = ['registry.access.redhat.com', 'registry.fedoraproject.org', 'docker.io']
+```
+
+Or, edit the system-wide registries configuration file [registries.conf(5)](https://github.com/containers/image/blob/master/docs/containers-registries.conf.5.md)
+
+#### Recommended - Use systemd cgroups.
+
+By default, CRI-O uses cgroupfs as a cgroup driver. However, we recommend using systemd as a cgroup driver. You can change your cgroup driver in crio.conf:
+
+```
+cgroup_driver = "systemd"
+```
+
+#### Optional - Modify verbosity of logs in /etc/crio/crio.conf
+
+Can modify the `log_level` field in `/etc/crio/crio.conf` to change the verbosity of
+the logs.
+Options are fatal, panic, error (default), warn, info, and debug.
+
+```
+log_level = "info"
+```
+
```

Comment:
I think we should add another Optional here showing how we can modify the capabilities and sysctls enabled by default in cri-o. Since we have only a short list of capabilities enabled, users might want to know how they can enable more etc.

Reply:
@umohnani8 having never done this myself, can you spin up a little tutorial in some pastebin?

Reply:
I think you can just state that the default capabilities and sysctls can be modified in crio.conf and just past the default list of capabilities we have currently.

Reply:
Added!

Reply:
Awesome, thanks!